cedar-policy/Cargo.toml

```diff
@@ -59,14 +59,24 @@ crate_type = ["rlib", "cdylib"]
 cool_asserts = "2.0"
 criterion = "0.5"
 globset = "0.4"
-cedar-policy-core = { version = "=4.0.0", features = ["test-util"], path = "../cedar-policy-core" }
+cedar-policy-core = { version = "=4.0.0", features = [
+    "test-util",
+], path = "../cedar-policy-core" }
```

Comment:
?

Reply:
I am not sure why my editor made this format, will delete